### PR TITLE
US26 user settings

### PR DESCRIPTION
**Pull request by wonecode**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/Form/ChangePasswordType.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                'constraints' => [
+                    new NotBlank([
+                        'message' => 'Entrez votre mot de passe',
+                    ]),
+                    new Length([
+                        'min' => 8,
+                        'minMessage' => 'Un mot de passe doit contenir au minimum {{ limit }} caractères ',
+                        // max length allowed by Symfony for security reasons
+                        'max' => 4096,
+                    ]),
```
Comment (LucyMerlier) [inline src/Form/ChangePasswordType.php]: you already put validations as annotations in your entity file

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/Repository/ChangePasswordRepository.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.exampleField = :val')
+            ->setParameter('val', $value)
+            ->getQuery()
+            ->getOneOrNullResult()
+        ;
+    }
+    */
+}
```
Comment (LucyMerlier) [inline src/Repository/ChangePasswordRepository.php]: this file is useless

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**